### ISSUE TITLE
Template Provisioning - Unable to set the Term Default Values to folders which contains French Letters.

### ISSUE THREAD

**Issue by swapnilekhe** [pnp/pnpframework#231]
I am using version 1.3.0 Pnp.Framework to apply Xml Template to SharePoint Site which contains the Folders/Inner Folders associated Document List and its default Term values as GUID (or with format e.g. 8;#{TermName}|{GUID}).  Here I am unable to apply the Term default values to folders which contains French letters. 

But able to apply if the Folder Name or Default Term Name contains plain English letters.

e.g. Folder Name or Term Name contains letters as rché, ôture, d'auto 

Please help if anyone faced such issue or any kind of resolution. Thanks.

**Comment by jackpoz** [comment]
could you please provide a template that can be applied to reproduce the issue ?

**Comment by swapnilekhe** [comment]
Please refer the attached sample template. You can see the folders against the Document Library list having below names in french for which we are unable to set the Term value through c# provisioning api.
1. Documentation de clôture - Projet
2. VD Financière

but able to set for the folder which has english name. e.g below.
1. Data principle

**Comment by jackpoz** [comment]
When applying the template I get an error "Invoke-PnPSiteTemplate: The field was found invalid:" which seems to happen when processing "{termsetid:PPMI-DEV:Type document}" . Do we need to add some Taxonomy structure to be able to apply the template ?

**Comment by swapnilekhe** [comment]
yes, we need to have a termset that we can use to set default value of the folders. Please refer attached another simple template where you can replace the default value guid and update the TermGroup with Termset name
[New_Sample_Template.txt](https://github.com/pnp/pnpframework/files/6570279/New_Sample_Template.txt)
